styles/mixins/themes.scss

```diff
@@ -109,7 +109,7 @@ $scopedTheme: null;
             $scopedTheme: () !global;
             @each $variableName, $variableValue in $themeMap {
                 // Merge each key-value pair into the theme specific map
-                $scopedTheme: map-merge($scopedTheme, ($variableName: $variableValue)) !global;
+                $scopedTheme: map.merge($scopedTheme, ($variableName: $variableValue)) !global;
```

Comment:
It looks like both of these are technically correct according to https://sass-lang.com/documentation/modules/map#merge

The linter seems to prefer the `map.merge` version as do the examples on https://sass-lang.com/documentation/values/maps#add-to-a-map

Reply:
Oh and there's also this - https://css-tricks.com/introducing-sass-modules/